pubsub/cloud-client/quickstart/pub.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START pubsub_quickstart_pub_all]
+import argparse
+import time
+# [START pubsub_quickstart_pub_deps]
+from google.cloud import pubsub_v1
+# [END pubsub_quickstart_pub_deps]
+
+
+def get_callback(api_future, data):
+    """Wraps message data in the context of the callback function."""
+
+    def callback(api_future):
+        try:
+            print("Published message {} now has message ID {}".format(
+                data, api_future.result()))
+        except Exception:
+            print("A problem occurred when publishing {}: {}\n".format(
+                data, api_future.exception()))
+            raise
+    return callback
+
+
+def pub(project_id, topic_name):
+    """Publishes a message to a Pub/Sub topic."""
+    # [START pubsub_quickstart_pub_client]
+    # Initializes the Publisher client
```

Comment:
Nit: When explaining what the code does (as opposed to "why"), write instructions like you are talking to the developer and telling them the code to write.

Prefer: Initialize a Publisher client

Likewise for the other comments in these samples.

Reply:
Done.